include/LArSED.h

```diff
@@ -205,51 +205,51 @@ void LArSED::Init(TTree *tree)
     // (once per file to be processed).
 
     // Set object pointer
-    nuPDG = 0;
```

Comment:
Not critically important for this PR, but the member variables are not in the Pandora style. It would also be preferable to intialise the variables in the constructor initialisation list

Reply:
This has been modified accordingly, as well as the LArSP and LArSPMC header files that have a similar structure.